addons/crm/views/crm_lead_views.xml

```diff
@@ -731,7 +716,108 @@
                             'default_use_template': True,
                             'default_template_id': ref('crm.email_template_opportunity_mail'),
                         }"
-                />
+        />
+
+        <act_window name="Send email"
+                res_model="mail.compose.message"
+                binding_model="crm.lead"
+                binding_views="list"
+                view_mode="form"
+                target="new"
+                id="crm.action_lead_single_mail"
+                context="{
+                            'default_composition_mode': 'comment',
+                            'default_email_to':'{($object.email or \'\')|safe}',
+                            'default_use_template': True,
+                            'default_template_id': ref('crm.email_template_opportunity_mail'),
+                        }"
+        />
+
+        <record id="crm_case_tree_view_oppor" model="ir.ui.view">
```

Comment:
In order to lessen the diff you should keep this one were it was defined first (l 583), otherwise we are a bit lost with the changes